docs/test/deploy-from-settings.md

```diff
@@ -0,0 +1,47 @@
+|Port                        |0                 |Positive integer                  |Negative integer|Not integer     |Not a number             |                     |                |       |
+|Target Port                 |0                 |Positive integer                  |Negative integer|Not integer     |Not a number             |                     |                |       |
+|Protocol                    |TCP               |UDP                               |                |                |                         |                     |                |       |
+|Expose service externally   |Checked           |Not checked                       |                |                |                         |                     |                |       |
+|Description                 |Empty             |Correct value                     |Max-length      |Validation error|                         |                     |                |       |
+|Key(Labels)                 |Empty             |Correct value                     |Max-length      |Validation error|Existing key             |                     |                |       |
+|Value(Labels)               |Empty             |Correct value                     |Max-length      |Validation error|                         |                     |                |       |
+|Namespace                   |Existing namespace|Select "Create a new namespace..."|                |                |                         |                     |                |       |
+|Image Pull Secret           |Existing secret   |Select "Create a new secret..."   |                |                |                         |                     |                |       |
+|CPU requirement(cores)      |0                 |Positive integer                  |Negative integer|Not integer     |Not a number             |                     |                |       |
+|Memory requirement(MB)      |0                 |Positive integer                  |Negative integer|Not integer     |Not a number             |                     |                |       |
+|Run command                 |Empty             |Correct value                     |Max-length      |Validation error|                         |                     |                |       |
+|Run command arguments       |Empty             |Correct value                     |Max-length      |Validation error|                         |                     |                |       |
+|Run as privileged           |Checked           |Not checked                       |                |                |                         |                     |                |       |
+|Key(Environment variables)  |Empty             |Correct value                     |Max-length      |Validation error|Existing key             |                     |                |       |
```

Comment:
We should test if the environment variable is really set in the container (at least once). Add a comment

BTW: env variables are not displayed on the details page, maybe they should